esphome/components/bmp280_spi/sensor.py

```diff
@@ -0,0 +1,21 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import spi
+from ..bmp280_base import to_code_base, CONFIG_SCHEMA_BASE
+
+AUTO_LOAD = ["bmp280_base"]
+DEPENDENCIES = ["spi"]
```

Comment:
Not sure why you removed as codeowner on the actual new component you are adding yourself...
```suggestion
CODEOWNERS = ["@ademuri"]
DEPENDENCIES = ["spi"]
```